pkgs/development/python-modules/json-home-client/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+# build inputs
+, typing-extensions
+, uri-template
+}:
+
+buildPythonPackage rec {
+  pname = "json_home_client";
+  version = "1.1.1";
+
+  disabled = pythonOlder "3.6";
+
+  #TODO: Figure out why Pypi doesn't host the source package
```

Comment:
```suggestion
```
We don't need to do that. We can fetch from the version control system.

Reply:
we can, yes, but then it requires this `postPatch` step that I'd rather not have